package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "brainstem-redux",
-  "version": "0.0.37",
+  "version": "0.1.0",
```

Comment:
I'm not sure this entailed a minor bump since it only affects the dev / test environments. But it doesn't really matter in the grand scheme of things.

Reply:
Less than `1.0.0`, semver rules are the wild west officially speaking. Definitely true that it's pretty minor, hence the version bump!